docs/usage/supported_k8s_versions.md

```diff
@@ -0,0 +1,10 @@
+# Supported Kubernetes Versions
+
+We strongly recommend using `etcd-druid` with the supported kubernetes versions, published in this document.
+The following is a list of kubernetes versions supported by the respective `etcd-druid` versions.
+
+| Etcd-druid version | Kubernetes version |
+|------|------|
+| >=0.20 | >=1.25 |
```

Comment:
This says druid of version >=0.20 doesn't support Kubernetes version 1.24. We are planning to support 1.24 in `Gardener` till the end of the year or later. Will the 0.20 release of the druid will come after the end of this year?